monai/transforms/utility/array.py

```diff
@@ -725,6 +729,8 @@ def __call__(
                 lines.append(f"Value range: (not a PyTorch or Numpy array, type: {type(img)})")
         if self.data_value if data_value is None else data_value:
             lines.append(f"Value: {img}")
+        if self.meta_info if meta_info is None else meta_info:
+            lines.append(f"Meta_Info: {MetaTensor(img).meta.__repr__()}")
```

Comment:
```suggestion
        if self.meta_info if meta_info is None else meta_info:
            lines.append(f"Meta_Info: {repr(MetaTensor(img).meta)}")
```
Unless you need to you should use the function rather than the special function. Perhaps we should check if `img` is a `MetaTensor` instance here and do nothing if it's not? Always converting to `MetaTensor` like this maybe isn't ideal. If you agree we would have:

```suggestion
        if isinstance(img, MetaTensor) and (self.meta_info if meta_info is None else meta_info):
            lines.append(f"Meta_Info: {repr(img.meta)}")
```

Reply:
Sorry for the procrastination. Thanks for the suggestion, maybe I should change it to the following code:

```
if self.meta_info if meta_info is None else meta_info:
  metadata = getattr(img, "meta", None)
  lines.append(f"Meta_Info: {repr(metadata)}")
```

Does this look better?

Reply:
I would say that we shouldn't print anything if `img` is not a MetaTensor rather than imply it is and whose `meta` value is `None`. I would also write "Meta info" instead of "Meta_Info" in the string as well.

Reply:
You're right, but I think users have already set the argument to `meta_info=True` in `Datastat` or `Datastatd`. Perhaps we should give some feedback after checking. Would it be too verbose if we responded more explicitly, like `Meta info: (input is not a MetaTensor)`?
```
if self.meta_info if meta_info is None else meta_info:
  metadata = getattr(img, "meta", "(input is not a MetaTensor)")
  lines.append(f"Meta info: {repr(metadata)}")
```

Reply:
Agree, you can add something like this. What do you think? @ericspod